test/controllers/forms/QuestionnaireDiversityFormSpec.scala

```diff
@@ -29,7 +29,7 @@ class QuestionnaireDiversityFormSpec extends BaseSpec {
       actualData mustBe expectedData
```

Comment:
filanem in class and test don't match. Need to be renamed to QuestionnaireDiversityInfoFormSpec?